.github/workflows/test.yml

```diff
@@ -25,6 +25,7 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get install libsane-dev
+          python3 -m pip install setuptools
```

Comment:
We shouldn't assume 3.12 users have setuptools installed, and should specify the build backend using pyproject.toml 

https://packaging.python.org/en/latest/tutorials/packaging-projects/

Reply:
Ok, I've replaced this with pyproject.toml